docs/docs/manual/exploring.md

```diff
@@ -1,102 +1,346 @@
+
+We can also explore our data using the population information. In this case, because population is a number, we can create a numeric facet. This will give us the ability to explore by range rather than by exact matching values. 
+
+With the numeric facet, we are given a scale from the smallest to the largest value in the column. We can drag the range minimum and maximum to narrow the results. In this case, if we narrow down to only cities with more than 20 million in population, we get 3 matching rows out of the original 10. 
+
+When you look at the facet display of countries, you should see a smaller list with a reduced count: OpenRefine is now displaying the facets of the 3 matching rows, not the total dataset of 10 rows. 
 
-(Types of Facet views and their common controls such as dragging Facets, organizing, sub elements like include/exclude, edit, sort by, change, reset, etc.)
+We can combine these facets - say, by narrowing to only the Chinese cities with populations greater than 20 million - simply by clicking in both. You should see 2 matching rows for both these criteria. 
 
+#### Things to know about facets
+
+When you have facets applied, you will see “matching rows” in the [project grid header](running#project-grid-header). If you press “Export” and copy your data out of OpenRefine while facets are active, you will only export the matching rows, not all the rows in your project. 
+
+OpenRefine has several default facets, which you’ll learn about below. The most powerful facets are the ones designed by you - custom facets, written using [expressions](expressions) to transform the data behind the scenes and help you narrow down to precisely what you’re looking for. 
+
+Facets are not saved in the project along with the data. But you can save a *permalink* to the current state of the application. Find the "permalink" next to the project’s name.
+
+You can modify any facet by clicking the “change” button to the right of the column name in the facet sidebar.
+
+Facet boxes that appear in the sidebar can be resized and rearranged. You can drag and drop the title bar of each box to reorder them, and drag on the bottom bar of text facet boxes. 
 
 ### Text facet
 
+A text facet can be generated on any column with the “text” data type. Select the column dropdown and go to “Facet” > “Text facet”. The created facet will be sorted alphabetically, and can be sorted by count. 
+
+A text facet is very simple: it takes the total contents of the cells of the column in question and matches them up. It does no guessing about typos or near-matches. 
+
+You can edit any entry that appears in the facet display, by hovering over the facet and clicking the “edit” button that appears. You can also automate the cleanup of facets by using clustering, with the “Cluster” button displayed within the facet window. 
```

Comment:
The editing part might be worth expanding, it's a powerful feature that is not so easy to discover. It would be worth explaining that this will change all the corresponding cell values: it is an effective way to clean up values in a column. You could add a link to the documentation about clustering (to be written) which does that at a larger scale.

Reply:
We internally call it a "mass edit" link, because it can massively edit from/to all the values in a column as Antonin says.

Reply:
I added a sentence or two. I can probably say more here so I'll leave this open for now. Current text: 

> You can edit any entry that appears in the facet display, by hovering over the facet and clicking the “edit” button that appears. This will mass-edit every matching cell in your data. This is a great way to fix typos, whitespace, and other issues that may be affecting the way facets appear. You can also automate the cleanup of facets by using [clustering](transforming#cluster-and-edit), with the “Cluster” button displayed within the facet window.